### PR TITLE
Add `plugin-nvim`

### PR DESCRIPTION
**Pull request by tdroxler**
Please don't hesitate to correct my poor english :)

Resolves: #164

### CODE REVIEW

**Review by simerplaha — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,16 +4,42 @@ Language server for Ralph.
 
 Currently supports text document level events and diagnostics.
 
+# Installation
```
Comment (simerplaha) [inline README.md:7]: Please consider moving this after `Install VSCode plugin` and `Install Neovim plugin`. Most devs probably just want to download plugins for their IDEs.

**Comment by simerplaha** [comment]
> Please don't hesitate to correct my poor english :)

Minor grammatical things, can be resolved later.

**Comment by simerplaha** [comment]
> Please don't hesitate to correct my poor english :)

Very minor, no biggie at all, can be resolved later. It is written well 👍🏼

File: README.md (hunk 1 above)
Comment (simerplaha) [inline README.md:7]: WDYT of moving this after `Install VSCode plugin` and `Install Neovim plugin`? Most devs probably just want to download plugins for their IDEs.

Comment (simerplaha) [inline README.md:7]: WDYT of moving this after `Install VSCode plugin` and or within `Install Neovim plugin`? Most devs probably just want to download plugins for their IDEs and this installation step is not required for VSCode. But it is required for VIM.

Comment (simerplaha) [inline README.md:7]: Feel free to ignore this: WDYT of moving this step after `Install VSCode plugin` or within `Install Neovim plugin`? Most devs probably just want to download plugins for their IDEs, and this installation step is not required for VSCode but is required for Neovim.

Comment (tdroxler) [reply README.md:7]: I'll move it inside neovim section for now

**Pull request merged**